.github/workflows/gh-pages.yaml

```
@@ -6,8 +6,20 @@ on:
       - master
       - mainnet
       - 'forks/**'
+    paths:
+      - 'src/ethereum/**'
+      - 'static/**'
+      - '.github/workflows/gh-pages.yaml'
```

Should we add:

```suggestion
      - '.github/workflows/gh-pages.yaml'
      - 'uv.lock'
      - 'src/ethereum_spec_tools/docc.py'
      - 'src/ethereum_spec_tools/forks.py'
```

?

---

Added thanks!